templates/ansible/roles/nginx/templates/nginx_puma.conf.j2

```diff
@@ -0,0 +1,34 @@
+# {{ ansible_managed }}
+
+upstream app {
+  server {{ puma_bind_path }} fail_timeout=0;
+}
+
+server {
+  listen 80;
+  listen 443 ssl;
+
+  sendfile off;
+
+  server_name {{ server_name }};
+
+  ssl_certificate {{ ssl_certificate_path }};
+  ssl_certificate_key {{ ssl_certificate_key_path }};
+  ssl_protocols  TLSv1 TLSv1.1 TLSv1.2;  # don’t use SSLv3 ref: POODLE
```

Comment:
@RDrop I am not sure if SSL is supported currently, let's stick with http for now